packages/core/src/modules/credentials/formats/indy/IndyCredentialFormatService.ts

```diff
@@ -529,17 +529,14 @@ export class IndyCredentialFormatService extends CredentialFormatService {
     }
     return false
   }
-  public async deleteCredentialById(
-    credentialRecord: CredentialExchangeRecord,
-    options: DeleteCredentialOptions
-  ): Promise<void> {
+  public async deleteCredentialById(credentialRecord: CredentialExchangeRecord): Promise<void> {
     const indyCredential = credentialRecord.credentials.filter((binding) => {
       return binding.credentialRecordType == CredentialFormatType.Indy
     })
     if (indyCredential.length != 1) {
       throw new AriesFrameworkError(`Could not find Indy record id for credential record ${credentialRecord.id}`)
     }
-    if (options?.deleteAssociatedCredentials && indyCredential[0].credentialRecordId) {
+    if (indyCredential[0].credentialRecordId) {
       await this.indyHolderService.deleteCredential(indyCredential[0].credentialRecordId)
     }
   }
```

Comment:
I see you didn't add this in this PR, but I think we can make this a lot simpler by just passing the actual credentialRecordId  form the service. We already loop through the credentials there and find the correct format service (see my last suggestion on how to call this method from there, only the id is passed.). The body of this function will then be as simple as:

```ts
  public async deleteCredentialById(credentialRecordId: string): Promise<void> {
    await this.indyHolderService.deleteCredential(credentialRecordId)
  }
```